tests/integration_tests/modules/test_set_password.py

```diff
@@ -127,14 +130,16 @@ def test_explicit_password_set_correctly(self, class_client):
 
     def test_shadow_expected_users(self, class_client):
         """Test that the right set of users is in /etc/shadow."""
-        shadow = class_client.read_from_file("/etc/shadow")
+        shadow = class_client.read_from_file("/etc/shadow", use_sudo=True)
         for user_dict in USERS_DICTS:
             if "name" in user_dict:
                 assert "{}:".format(user_dict["name"]) in shadow
 
     def test_sshd_config(self, class_client):
         """Test that SSH password auth is enabled."""
-        sshd_config = class_client.read_from_file("/etc/ssh/sshd_config")
+        sshd_config = class_client.read_from_file(
+            "/etc/ssh/sshd_config", use_sudo=True
+        )
```

Comment:
no need for use_sudo here: ls -l /etc/ssh/sshd_config world readable -rw-r--r--
```suggestion
 sshd_config = class_client.read_from_file("/etc/ssh/sshd_config")
```